RecoMuon/MuonIdentification/src/classes.h

```diff
@@ -0,0 +1,23 @@
+#include "DataFormats/Common/interface/Wrapper.h"
+
+//Add includes for your classes here
+#include "RecoMuon/MuonIdentification/interface/VersionedMuonSelector.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+
+#include "DataFormats/PatCandidates/interface/Muon.h"
+
+#include "PhysicsTools/SelectorUtils/interface/MakePyVIDClassBuilder.h"
+#include "PhysicsTools/SelectorUtils/interface/MakePtrFromCollection.h"
+#include "PhysicsTools/SelectorUtils/interface/PrintVIDToString.h"
+
+namespace RecoMuon_MuonIdentification {
```

Comment:
FWLite distribution doesn't have this package, IIRC.
This implies that to use the code the full release should be used.
This is probably true about all of this ID framework.